tests/TestCase/Filesystem/FolderTest.php

```diff
@@ -277,11 +277,11 @@ public function testOperations()
         $this->assertTrue($result);
 
         $copy = TMP . 'tests' . DS . 'test_folder_copy';
-        $result = $Folder->copy($copy);
+        $result = $Folder->copy(['to' => $copy]);
```

Comment:
This is not a convenient usage. In most cases one would just specify destination path without any options. So changing the method signature to `copy(string $dest, array $options)` would be better.